termwiz/Cargo.toml

```diff
@@ -27,6 +27,7 @@ regex = "1"
 semver = "0.9"
 serde = {version="1.0", features = ["rc", "derive"], optional=true}
 terminfo = "0.7"
+thiserror = "1.0.23"
```

Comment:
I prefer `thiserror = "1.0"` rather than specifying the minor number here.
The `Cargo.lock` file tracks the resolved versions.